tests/integration/tier1/destructive/error-after-ponr/test_error_after_ponr.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+from conftest import TEST_VARS
+
+
+@pytest.mark.test_error_after_ponr
+def test_error_after_ponr(convert2rhel, shell):
+    """
+    Verify the improved explanatory log message is displayed to the user when a fail occurs after the point of no return
+
+    And an exit code is 1
```

Comment:
just indention and wording
```suggestion
    Verify the improved explanatory log message is displayed to the user when a fail occurs
    after the point of no return. Also verify that the exit code is 1.
```